docs/includes/deploy-updates-that-require-admin-consent.md

```diff
@@ -0,0 +1,7 @@
+## Deploy updates
+
+When you add features or fix bugs in your add-in you'll need to deploy the updates. If your add-in is deployed by one or more admins to their organizations, some manifest changes will require the admin to consent to the updates. Users will be blocked from the add-in until consent is granted. We recommend you test deployment behavior on a staging site before deploying updates live. The following manifest changes will require the admin to consent again.
```

Comment:
Some minor grammar/phrasing suggestions.
```suggestion
When you add features or fix bugs in your add-in, you'll need to deploy the updates. If your add-in is deployed by one or more admins to their organizations, some manifest changes require the admin to consent to the updates. Users will be blocked from the add-in until consent is granted. We recommend you test on a staging site before deploying live updates. The following manifest changes require the admin to consent again.
```

Reply:
Removed this sentence for now until I can confirm if the testing step is actually needed.